templates/make_single_pdf_page.R

```diff
@@ -0,0 +1,53 @@
+#------------------------------------------------------
```

Comment:
as-is this file **will not** be included in the R package installation.

We can fix that though.

However, a larger change will be needed as currently `ihme.covid` is being installed in a **random directory on a nodes local `/tmp` filesystem** which means even if we bundle this file it'll be missing if the job executes on a different node.

I also know how to fix that. I'll need to do more work than I can before my next meeting but basically

- It's time to install `ihme.covid` on the IHME file system and manually update it when changes are released
- then we reference that global installation

Reply:
I actually don't think this file is needed - I moved this to a Hub page because it's just an example. But I would support a global install!